monzo/utils.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+def parse_datetimes(transactions):
```

Comment:
Might it be better to turn this into `parse_datetime` which takes a single transaction?

We could then perform list comprehension to recover `parse_datetimes` while allowing us to parse a single transaction without wrapping it in a list.

Reply:
Hey. We could certainly do that. I did it this way because there is currently no way to fetch a single transaction (on which note, we should really add a `get_transaction()` endpoint). Not sure `parse_datetime` works as a name, however, because a single transaction contains multiple datetimes. Let me push another commit.

Reply:
Ah, agreed on the name. I forgot about the multiple keys. New commit looks good.